### PR TITLE
Enrich Indicators with IntSights from InsightIDR

### PR DESCRIPTION
**Pull request by nhurd-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [x] Workflow verified as working in a test
- [x] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [x] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] Workflow includes a detailed Description
- [x] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] Triggers, Decisions, and other workflow steps include descriptions where needed
- [x] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [x] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [x] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [x] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

```
### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

![WorkflowView1](https://user-images.githubusercontent.com/74786214/148594207-aacae6b7-b46b-4e92-8de3-37ff7877540d.png)
![WorkflowView2](https://user-images.githubusercontent.com/74786214/148594208-a640909d-4c44-4f13-8ffd-2f87553117e3.png)
![WorkflowView3](https://user-images.githubusercontent.com/74786214/148594209-90be1830-a15d-4929-9e44-c4c7351bbf9d.png)
![WorkflowView4](https://user-images.githubusercontent.com/74786214/148594210-a336782e-25d8-4abb-8090-f530547fe525.png)
![WorkflowView5](https://user-images.githubusercontent.com/74786214/148594211-06ecde04-a1a4-4b6f-8854-e8b4b7897d4f.png)

#### Job

Replaced with every permutation of Artifact screenshot instead

#### Artifact

![DomainLookupResults](https://user-images.githubusercontent.com/74786214/148594240-6d1cebf0-fc26-4493-9bf8-504561cec5af.png)
![HashLookupResults](https://user-images.githubusercontent.com/74786214/148594241-3c9f4b4d-2f02-4ee7-92d6-a4776413e549.png)
![IPLookupResults](https://user-images.githubusercontent.com/74786214/148594242-68daae82-6747-4a36-af82-078993cbf15f.png)
![URLLookupResults](https://user-images.githubusercontent.com/74786214/148594243-a743a0bd-3bff-473f-a717-9810e16164cd.png)

```

### CODE REVIEW

File: workflows/Enrich_Indicators_with_IntSights_from_InsightIDR/workflow.spec.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+name: Enrich_Indicators_with_IntSights_from_InsightIDR
+title: "Enrich Indicators with IntSights from InsightIDR"
+description: This workflow enriches an IDR alert by performing a lookup on all domains, hashes, URLs and IPs in the investigation with Rapid7 IntSights.
+version: 2.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: []
+  keywords: [Data Enrichment, Threat Detection & Response, Enrichment, Hash, URL, Domains, IP Address, Rapid7
```
Comment (pmara-r7) [inline workflows/Enrich_Indicators_with_IntSights_from_InsightIDR/workflow.spec.yaml:12]:
```suggestion
  keywords: [Data Enrichment, Threat Detection & Response, Enrichment, Hash, URL, Domains, IP Address, Rapid7]
```

**Review by pmara-r7 — CHANGES_REQUESTED** [review]

File: workflows/Enrich_Indicators_with_IntSights_from_InsightIDR/workflow.spec.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+name: Enrich_Indicators_with_IntSights_from_InsightIDR
+title: "Enrich Indicators with IntSights from InsightIDR"
+description: This workflow enriches an IDR alert by performing a lookup on all domains, hashes, URLs and IPs in the investigation with Rapid7 IntSights.
+version: 2.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: []
+  keywords: [Data Enrichment, Threat Detection & Response, Enrichment, Hash, URL, Domains, IP Address, Rapid7]
```
Comment (pmara-r7) [inline workflows/Enrich_Indicators_with_IntSights_from_InsightIDR/workflow.spec.yaml:12]:
```suggestion
  use_cases: [Data Enrichment, Thread Detection & Response]
  keywords: [Data Enrichment, Threat Detection & Response, Enrichment, Hash, URL, Domains, IP Address, Rapid7, parameters]
```

File: workflows/Enrich_Indicators_with_IntSights_from_InsightIDR/workflow.spec.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+products: ["insightconnect"]
+name: Enrich_Indicators_with_IntSights_from_InsightIDR
+title: "Enrich Indicators with IntSights from InsightIDR"
+description: This workflow enriches an IDR alert by performing a lookup on all domains, hashes, URLs and IPs in the investigation with Rapid7 IntSights.
+version: 2.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: [Data Enrichment, Thread Detection & Response]
```
Comment (pmara-r7) [inline workflows/Enrich_Indicators_with_IntSights_from_InsightIDR/workflow.spec.yaml:11]:
```suggestion
  use_cases: ['data_enrichment', 'threat_detection_and_response']
```

**Review by blaxminarayan-r7 — CHANGES_REQUESTED** [review]

**Comment by mcochran-r7** [comment]
closed due to age

**Pull request closed without merging**